### PR TITLE
Refactor all specs

### LINKED ISSUE #160: refactor "all" specs
apply https://www.betterspecs.org/ rules

### PR DESCRIPTION
**Pull request by garciajordy**
# Rspec

**Refactored all the specs.**

### Co-authored-by: Daniel Nottingham <danielnottingham@users.noreply.github.com>
fix #160

### CODE REVIEW

**Review by edimossilva — CHANGES_REQUESTED** [review]
@garciajordy and @danielnottingham , well done, can you please add the rule to rubocop that only allows 1 expectation per `it`.

File: backend/spec/system/question_crud_spec.rb [hunk 1]
```diff
@@ -24,10 +24,9 @@
           find('.option', text: question_type.name).click
         end
 
-        it 'shows all surveys' do
+        it 'expect to show admin and teacher surveys' do
           find('label', text: 'Survey').click
-          expect(page).to have_selector('.option', text: survey_admin.name)
-          expect(page).to have_selector('.option', text: survey_teacher.name)
+          expect(page).to have_selector('.option', text: survey_admin.name && survey_teacher.name)
```
Comment (garciajordy) [inline backend/spec/system/question_crud_spec.rb:29]: Did you know that this was possible? Just found out by myself xD So cool!

Comment (edimossilva) [reply backend/spec/system/question_crud_spec.rb:29]: I am not sure if this works, can you try something like `expect(page).to have_selector('.option', text: survey_admin.name && survey_teacher)` and tell me if it also pass?

**Review by edimossilva — APPROVED** [review]
LGTM! nice!

**Pull request merged**